lib/sandbox/sandbox.go

```diff
@@ -23,13 +23,13 @@ import (
 // NetNs handles data pertaining a network namespace
 type NetNs struct {
 	sync.Mutex
-	ns       ns.NetNS
+	netNS    ns.NetNS
 	symlink  *os.File
 	closed   bool
 	restored bool
 }
 
-func (ns *NetNs) symlinkCreate(name string) error {
+func (nns *NetNs) symlinkCreate(name string) error {
```

Comment:
Can't we call this netns?

Reply:
it needs to be public, since it's used outside this package, right?

Reply:
unless you mean the internal reference of the function?

Reply:
Yes the internal name.

Reply:
done